frontend/src/components/WorldMap.jsx

```diff
@@ -16,12 +16,15 @@ const WorldMap = ({ countries, onCountrySelect }) => {
     const confirmedText = country.properties.confirmedText;
     layer.bindPopup(`${name} ${confirmedText}`);
     layer.on("click", function (e) {
-      onCountrySelect(e.target.feature.properties.iso_code);
+      onCountrySelect(
+        e.target.feature.properties.name,
+        e.target.feature.properties.confirmedText
```

Comment:
I wonder whether we should rename `confirmed` and `confirmedText` to eg. `totalCasesPerMillion` and `totalCasesPerMillionText`?

Reply:
And done :)